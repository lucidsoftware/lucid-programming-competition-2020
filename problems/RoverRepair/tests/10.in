31
8
2
12
4
5
11
13
1
3
empty
7
10
empty
empty
14
0
empty
empty
empty
empty
empty
6
empty
9
empty
empty
empty
empty
empty
empty
empty
