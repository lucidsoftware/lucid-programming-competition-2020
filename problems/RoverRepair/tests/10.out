Swap nodes 2 and 4
